### PR TITLE
Feature/gometa tiny

### PR DESCRIPTION
**Pull request by ruseinov**
work on #230 
#trivial

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -32,6 +35,12 @@ test:
 	go vet -mod=readonly  ./...
 	go test -mod=readonly -race ./...
 
+lint:
+ifndef LINT
+	./scripts/golangci-lint.sh -b $(shell go env GOPATH)/bin v1.17.1
```
Comment (husio) [inline Makefile]: I just thought, is there a docker container available, that we could use instead?
We are using a docker container to compile protocol buffer files and it works great.

File: .golangci.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    check-shadowing: false
+    settings:
+      printf:
+        funcs:
+          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Infof
+          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Warnf
+          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Errorf
+          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Fatalf
+  lll:
+    line-length: 140
```
[the remark this answers is not in the document]
Comment (ruseinov) [reply .golangci.yaml]: Enforcing line length is good, but currently I don't think this is enabled. 
I guess this linter is switched off by default.

File: Makefile (hunk 1 above)
Comment (ruseinov) [reply Makefile]: That is a good call for a future PR, I'd like to do that for sure. Saves us from the necessity of having to download 3rd party stuff from github.

File: .golangci.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (ruseinov) [reply .golangci.yaml]: I actually enforce my lines at about 120 in the IDE. I find it nicer to read, especially if you need to open this in github/two windows side by side.

I guess we'd have to have some sort of proposal about what should be enabled and with which options.

File: Makefile (hunk 1 above)
Comment (ethanfrey) [reply Makefile]: Actually, I would love to see this in a Docker image. It makes all tools much simpler.

After looking at that code, I don't see why not. Even a Dockerfile in this repo.
I will try that

Comment (ethanfrey) [reply Makefile]: Or what about https://hub.docker.com/r/golangci/golangci-lint/tags ?

Comment (ethanfrey) [reply Makefile]: I think it is fair to fit that in this PR

**Pull request merged**